packages/editor-ui/src/store.ts

```diff
@@ -818,6 +847,10 @@ export const store = new Vuex.Store({
 			return state.workflow.settings;
 		},
 
+		workflowTags: (state): string[] => {
+			return (state.workflow.tags as string[]) || [];
```

Comment:
Same comment as above regarding defaulting to empty array.

Reply:
it will always have a value. maybe these extra checks are unnecessary